### PR TITLE
Update content

### PR DESCRIPTION
**Pull request by Zmoki**

### CODE REVIEW

**Comment by Zmoki** [comment]
[Effects Tab: выпустить новую версию](https://trello.com/c/0jmpc0h2/267-effects-tab-%D0%B2%D1%8B%D0%BF%D1%83%D1%81%D1%82%D0%B8%D1%82%D1%8C-%D0%BD%D0%BE%D0%B2%D1%83%D1%8E-%D0%B2%D0%B5%D1%80%D1%81%D0%B8%D1%8E)

File: CHANGELOG.md [hunk 1]
```diff
@@ -22,26 +23,33 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 ## 1.0.3 - 2017-08-23
 
 ### Fixed
+
 * Fix links to uploadcare site in README.
```
Comment (dayton1987) [inline CHANGELOG.md:27]: Fixed references to the Uploadcare site in README.

File: CHANGELOG.md [hunk 2]
```diff
@@ -22,26 +23,33 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
...
 
 ## 1.0.2 - 2017-08-21
 
 ### Added
+
 * Uploadcare logo into README.
```
Comment (dayton1987) [inline CHANGELOG.md:33]: Uploadcare logo to README.

File: CHANGELOG.md [hunk 3]
```diff
@@ -22,26 +23,33 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
...
 
 ## 1.0.1 - 2017-08-21
+
 Initial public release for npm.
 
 ### Fixed
+
 * Fix info about version in README.
```
Comment (dayton1987) [inline CHANGELOG.md:41]: Unclear

File: CHANGELOG.md [hunk 4]
```diff
@@ -22,26 +23,33 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
...
 
 ### Added
+
 * README section about install with npm.
 * README section about output value.
```
Comment (dayton1987) [inline CHANGELOG.md:46]: Unclear, ... section about `<method>` output value

File: README.md [hunk 5]
```diff
@@ -6,15 +6,11 @@
          alt="">
 </a>
 
-Effects Tab is a [custom tab][uc-custom-tabs] for [Uploadcare Widget][uc-widget]
+Effects Tab is a [custom tab][uc-docs-widget-custom-tabs] for [Uploadcare Widget][uc-docs-widget]
 that replaces Preview Tab and adds image-editing capabilities to the widget.
```
Comment (dayton1987) [inline README.md]: Effects Tab is an Uploadcare Widget addon that allows for in-browser image editing on desktop and mobile. Technically, it's a custom widget tab that replaces Preview Tab.

File: README.md [hunk 6]
```diff
@@ -27,23 +23,29 @@ Effects Tab provides 9 effects for on-the-fly image editing on upload:
 crop, rotate, mirror, flip, blur, sharpen, enhance, grayscale and invert.
```
Comment (dayton1987) [inline README.md:25]: ... 9 effects for on-the-fly image editing in desktop or mobile browsers ...

File: README.md [hunk 7]
```diff
@@ -27,23 +23,29 @@ Effects Tab provides 9 effects for on-the-fly image editing on upload:
...
+* [Feedback](#feedback)
+
+<!-- tocstop -->
 
 ## How it works
 
-Image operations provided by Effects Tab are based on the capabilities of our [CDN API][uc-cdn].
-The tab outputs a CDN link holding your image [UUID][uc-uuid] and
-[image operations][uc-cdn-image-operations] applied by a user during the in-tab editing process.
+Image operations provided by Effects Tab are based on the capabilities of our [CDN API][uc-docs-cdn].
```
Comment (dayton1987) [inline README.md]: ... the capabilities of Uploadcare `[Image Processing][uc-feature-image-processing]` ...

File: README.md [hunk 8]
```diff
@@ -27,23 +23,29 @@ Effects Tab provides 9 effects for on-the-fly image editing on upload:
...
+The tab outputs a CDN link holding your image [UUID][uc-docs-uuid] and
+[image operations][uc-docs-cdn-image-operations] applied by a user during the in-tab editing process.
```
Comment (dayton1987) [inline README.md]: applied by a user while editing

File: package.json [hunk 9]
```diff
@@ -6,7 +6,7 @@
     "classPrefix": "uploadcare-tab-effects--"
   },
   "version": "1.1.0",
-  "description": "Effects Tab for Uploadcare Widget",
+  "description": "Effects Tab for Uploadcare Widget. Edit images in the browser.",
```
Comment (dayton1987) [inline package.json]: Edit images in desktop and mobile browsers.

**Review by dayton1987 — CHANGES_REQUESTED** [review]

**Pull request merged**